tests/templates/kuttl/external-access/20-install-redis.yaml

```diff
@@ -0,0 +1,12 @@
+---
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - script: >-
```

Comment:
Couldn't this be merged into 10-, so it can be done in parallel?

Reply:
I've left the product installs as they are (I can't think of anywhere else where we install products together in one step: we keep them discrete for good reason) but combined the asserts. Not sure what benefit this brings, TBH. f9c41bc.